web/app/AttributeFormatter.js

```diff
@@ -34,6 +34,19 @@ Ext.define('Traccar.AttributeFormatter', {
         return Ext.getStore('DistanceUnits').formatValue(value, Traccar.app.getPreference('distanceUnit'));
     },
 
+    intervalFormatter: function (value) {
```

Comment:
I could not find anything http://docs.sencha.com/extjs/6.0.1-classic/Ext.Date.html
Implementing own `formatFunction` won't be easier.
Microsoft suggesting the same algorithm  https://msdn.microsoft.com/en-us/library/ee532932(v=vs.94).aspx